### ISSUE TITLE
Handle upload form submit when no file selected

### ISSUE THREAD

**Issue by jcushman** [harvard-lil/perma#1709]
Two problems occur when the upload-file form is submitted without a file selected:

(1) In all browsers, the error message is "Upload failed. [object Object]"

(2) In some browsers (including Chrome 54 for Mac), we get an error thrown on the backend: "The format indicated 'application/x-www-form-urlencoded' had no available deserialization method".

**Comment by rebeccacremona** [comment]
Interesting. When I just did this locally, in place of [object Object] I got a full Django debug-style stacktrace.

Proposed solution in https://github.com/harvard-lil/perma/pull/1734.

Note: I don't know why the request is going out with Content-Type = "application/x-www-form-urlencoded; charset=UTF-8". That's not acceptable, according to the HTML5 spec. That seems like a funny thing for modern browsers to be doing.... Still digging.
